index.js

```diff
@@ -1,93 +1,105 @@
 "use strict";
 
+const semver = require("semver");
+const findDependencyVersion = require("./lib/find-dependency-version");
+
+const eslintVersion = semver.minVersion(
+  findDependencyVersion("eslint") || "0.0.0"
+);
+
 module.exports = {
-  rules: {
-    // The following rules can be used in some cases. See the README for more
-    // information. (These are marked with `0` instead of `"off"` so that a
-    // script can distinguish them.)
-    "arrow-body-style": 0,
-    curly: 0,
-    "lines-around-comment": 0,
-    "max-len": 0,
-    "no-confusing-arrow": 0,
-    "no-mixed-operators": 0,
-    "no-tabs": 0,
-    "no-unexpected-multiline": 0,
-    "prefer-arrow-callback": 0,
-    quotes: 0,
-    // The rest are rules that you never need to enable when using Prettier.
-    "array-bracket-newline": "off",
-    "array-bracket-spacing": "off",
-    "array-element-newline": "off",
-    "arrow-parens": "off",
-    "arrow-spacing": "off",
-    "block-spacing": "off",
-    "brace-style": "off",
-    "comma-dangle": "off",
-    "comma-spacing": "off",
-    "comma-style": "off",
-    "computed-property-spacing": "off",
-    "dot-location": "off",
-    "eol-last": "off",
-    "func-call-spacing": "off",
-    "function-call-argument-newline": "off",
-    "function-paren-newline": "off",
-    "generator-star": "off",
-    "generator-star-spacing": "off",
-    "implicit-arrow-linebreak": "off",
-    indent: "off",
-    "indent-legacy": "off",
-    "jsx-quotes": "off",
-    "key-spacing": "off",
-    "keyword-spacing": "off",
-    "linebreak-style": "off",
-    "multiline-ternary": "off",
-    "newline-per-chained-call": "off",
-    "new-parens": "off",
-    "no-arrow-condition": "off",
-    "no-comma-dangle": "off",
-    "no-extra-parens": "off",
-    "no-extra-semi": "off",
-    "no-floating-decimal": "off",
-    "no-mixed-spaces-and-tabs": "off",
-    "no-multi-spaces": "off",
-    "no-multiple-empty-lines": "off",
-    "no-reserved-keys": "off",
-    "no-space-before-semi": "off",
-    "no-spaced-func": "off",
```

Comment:
Looks like you forgot this one?

Reply:
Will put back, taking a different approach, see https://github.com/prettier/eslint-config-prettier/pull/120#issuecomment-544188602